crates/re_viewport/src/viewport.rs

```diff
@@ -249,6 +249,7 @@ impl<'a, 'b> Viewport<'a, 'b> {
                 blueprint.set_auto_layout(false, ctx);
             }
 
+            // TODO(#4687): We must be share not to mark edited if not edit has occurred
```

Comment:
Did you mean "We must be _sure_" ?

Reply:
:facepalm: yeah -- some day I'll learn how to type